examples/src/multiple_kms_cmk_regions.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""
+Example showing basic encryption and decryption of a value already in memory
+using multiple KMS CMKs in multiple regions.
+"""
+import aws_encryption_sdk
+
+
+def encrypt(kms_key_provider, source_plaintext):
+    """Encrypts source_plaintext with the key(s) in kms_key_provider"""
+    return aws_encryption_sdk.encrypt(source=source_plaintext, key_provider=kms_key_provider)
```

Comment:
IMO, separating out the encrypt and decrypt logic at this level doesn't really make sense; you're basically just replacing a call to `aws_encryption_sdk.encrypt` with a call to `encrypt`.

@ttjsu-aws could you elaborate on where you meant to ask that the separation be drawn?